src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $main-color;
```

Comment:
The font-family is not specified here. Ensure to add it for consistency across different browsers.